plugins/hydra_optuna_sweeper/example/mo.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
```

Comment:
for clarity, can you rename it (and corresponding config dirs etc) to multi-objective.py?

Reply:
Thank you for your suggestion. I applied it in https://github.com/facebookresearch/hydra/pull/1294/commits/65e8e7e9ecff6681c20ba4a8bb273f32f1c9957b.